app/tests/integration/tox.ini

```diff
@@ -6,7 +6,7 @@ passenv = *
 recreate = true
 commands =
     pip install -r ../requirements.txt
-    pytest --variables e2e_test_variables.json --verify-base-url --base-url https://nginx/ --driver Firefox . {posargs} -vvv
+    pytest --variables e2e_test_variables.json --verify-base-url --base-url https://nginx/ --driver Firefox . -k test_multi_pref {posargs} -vvv
```

Comment:
Is this `-k test_multipref` supposed to be committed?

Reply:
ahh frick